client/src/components/SignUp.js

```diff
@@ -1,13 +1,177 @@
 import React, { Component, Fragment } from 'react';
+import axios from 'axios';
+
+import * as S from './Login.style';
+
+class signUp extends Component {
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    nameError: '',
+    emailError: '',
+    passwordError: '',
+    isErrorName: false,
+    isErrorEmail: '',
+    isErrorPassword: false,
+  };
+
+  componentDidMount() {
+    axios.get('/checkCookie').then(({ data: { cookie } }) => {
+      if (cookie) {
+        const { history } = this.props;
+        history.push('/account/login');
```

Comment:
this should use a constant route (imported from constants, so I think it will something like LOGIN_URL